Sources/ShamTestHelpers/Extensions/XCTestCase+Stub.swift

```diff
@@ -1,24 +1,25 @@
 // Copyright © 2020 SpotHero, Inc. All rights reserved.
 
-#if !os(watchOS) && canImport(XCTest)
-    
+#if canImport(XCTest)
```

Comment:
It's odd to me that doing this wouldn't work. I wonder if there's a better way besides splitting the modules up?

I don't think this PR is a bad solution at all, I just think if we could solve for a conditional statement for including this extension, it simplifies everything downstream as well.

Reply:
Like, maybe just doing the equivalent of a `os(x386)` or whatever the code would be, would mean this is only imported if being built for simulators and running `XCTest`?

Just spitballing.

Reply:
@bdrelling Even if we limit it to just simulators, `targetEnvironment(simulator)`, building the main SpotHero project in Debug mode on a simulator will have errors - "Undefined symbol: __swift_FORCE_LOAD_$_XCTestSwiftSupport". I can get by these errors by setting "Enable Testing Search Paths" to "YES" in Build Settings on both SpotHeroData and SpotHero.

Another option is to set a specific flag on the test targets that validate on the if statement in the specific extensions, but I felt that it was cleaner to separate out the extensions into piecemeal packages for the testing targets instead.